Example/Example/Model/ContentType.swift

```diff
@@ -18,16 +18,11 @@ enum ContentType: String, Printable {
     }
     
     var image: UIImage {
-        let image =  UIImage(named: rawValue)!
+        let image =  UIImage(named: "content_\(rawValue.lowercaseString).png")!
```

Comment:
I think it makes sense not to use any patterns for image names because it doesn't make code clear.
Could plz create a constant and use it here
private let musicImageName = "content_music.png"
thanks